docs/reference/settings/audit-settings.asciidoc

```diff
@@ -18,10 +18,17 @@ Set to `true` to enable auditing on the node. The default value is `false`.
 `xpack.security.audit.outputs`::
 Specifies where audit logs are output. For example: `[ index, logfile ]`. The
 default value is `logfile`, which puts the auditing events in a dedicated
-`<clustername>_access.log` file on the node. You can also specify `index`, which
-puts the auditing events in an {es} index that is prefixed with
-`.security_audit_log`. The index can reside on the same cluster or a separate
-cluster.
+file named `<clustername>_audit.log` on each node.
+You can also specify `index`, which puts the auditing events in an {es} index
+that is prefixed with `.security_audit_log`. The index can reside on the same
+cluster or a separate cluster.
+
+For backwards compatibility reasons, the logfile contents are duplicated to
+another `<clustername>_access.log` file with an improved format.
```

Comment:
s/another/the

s/with an improved format/with the format for versions prior to 6.5.0.

Reply:
obviously... 
Got myself tangled up in all these words.
Thanks!